src/components/slide/slide-manager.jsx

```diff
@@ -292,6 +293,11 @@ function SlideManager({
   function handleMedia({ target }) {
     const fieldValue = target.value;
     const fieldId = target.id;
+
+    if (target.arrayName) {
```

Comment:
This should be changed following the implementation in https://jira.itkdev.dk/browse/AR-764